resources/prometheus/prometheus-rules.yaml

```diff
@@ -414,11 +414,11 @@ spec:
                 central:http_incoming_requests:not_5xx:rate10m
               + on (namespace) group_left(rhacs_instance_id, rhacs_org_id, rhacs_org_name, rhacs_cluster_name, rhacs_environment)
                 central:grpc_server_handled:server_available_code:rate10m
-              ) / (
+              ) / ((
                 central:http_incoming_requests:total:rate10m
               + on (namespace) group_left(rhacs_instance_id, rhacs_org_id, rhacs_org_name, rhacs_cluster_name, rhacs_environment)
                 central:grpc_server_started:total:rate10m
-              )
+              ) > 0)
```

Comment:
Did you check how `central:sli:availability` is affected if there is now a gap in `central:sli:error_rate`? Would there also be a gap in `central:sli:availability`?

Reply:
Yes, there would be a gap in those two metrics.  Is that a problem, though?  I tried to create a unit test that showed that the alerts wouldn't fire if this happened, but for some reason the grafana linter tests would fail when trying to parse the dashboard JSON.  Not sure how that was related, but ran out of time.

I went ahead and push the unit test even though I couldn't get the grafana linter to pass in my local box.

Reply:
Yeah, the problem is if the pod is not available (`central:sli:pod_ready=0`), no alerts would fire as well because data points are only present if `central:sli:error_rate` is defined.

Reply:
Perhaps it's possible to do a "left join" where the value of `central:sli:pod_ready` is taken if `central:sli:error_rate` is undefined?